docs/schedule/examples/pathways.de.md

```diff
@@ -0,0 +1,79 @@
+
+Zunächst werden die Lage des Bahnhofs und seine Eingänge in [stops.txt](../../reference/#pathwaystxt) definiert:
+
+[**stops.txt**](../../reference/#stopstxt)
+
+    stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station,wheelchair_boarding
+    12034,Waterfront Station,49.285687,-123.111773,1,,
+    90,Waterfront Station Stairs Entrance on Granville,49.285054,-123.114375,2,12034,2
+    91,Waterfront Station Escalator Entrance on Granville,49.285061,-123.114395,2,12034,2
+    92,Waterfront Station Elevator Entrance on Granville,49.285257,-123.114163,2,12034,1
+    93,Waterfront Station Entrance on Cordova,49.285607,-123.111993,2,12034,1
+    94,Waterfront Station Entrance on Howe,49.286898,-123.113367,2,12034,2
+
+In der obigen Datei bezieht sich der erste Datensatz auf den Standort des Bahnhofs, daher ist der `location_type` auf `1` gesetzt. Die anderen fünf Datensätze beziehen sich auf die drei Eingänge des Bahnhofs (fünf Datensätze sind erforderlich, da der Granville-Eingang eigentlich drei separate Eingänge hat, eine Treppe, eine Rolltreppe und einen Aufzug). Diese fünf Datensätze sind als Eingänge definiert, da der `location_type` auf `3` gesetzt ist.
+
+Außerdem wird die `stop_id` für Waterfront Station unter `parent_station` für die Eingänge aufgeführt, um sie mit der Station zu verknüpfen. Für die zugänglichen Eingänge ist `wheelchair_boarding` auf `1` und für die nicht zugänglichen auf `2` gesetzt.
+
+## Beschreibung der Treppen und Rolltreppen
+
+Der Eingang der Waterfront Station in der Granville Street verfügt über einen Aufzug, eine Rolltreppe und eine Treppe; die Eingänge sind als Knoten oben in [stops.txt](../../reference/#stopstxt) definiert. Um die Eingänge mit den inneren Bereichen des Bahnhofs zu verbinden, müssen zusätzliche Knoten in [stops.txt](../../reference/#stopstxt) unter der `parent_station` von Waterfront Station erstellt werden. In der folgenden Datei [stops.txt](../../reference/#stopstxt) werden generische Knoten`(location_type 3`) definiert, die dem unteren Ende der Treppe und der Rolltreppe entsprechen.
+
+[**stops.txt**](../../reference/#stopstxt)
+
+    stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station,wheelchair_boarding
+    ...
+    95,Waterfront Station Granville Stair Landing, 49.285169,-123.114198,3,12034,
+    96,Waterfront Station Granville Escalator Landing,49.285183,-123.114222,3,12034,
+
+<img class="center" src="../../../assets/pathways.png" width="700px"/>
```

Comment:
all three images 404 for all languages, wonder if this is a problem on my end or not, flagging because they rendered fine on the continuous stops page